cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/TestNativeImageCommandTest.java

```diff
@@ -50,35 +50,35 @@ public void setup() throws IOException {
         }
     }
 
-    @Test(description = "Test a valid ballerina project")
+    @Test(description = "Test a valid ballerina project", enabled = false)
```

Comment:
For me locally they are failing with some exception: https://pastebin.com/Jcbtdmge
And if I look into the build scan of the Pipeline on master then this whole testclass is not executed, which is quite strange, since I couldn't find where it is disabled. See also https://scans.gradle.com/s/mtlatsn45jzwc/tests/overview?collapse-all&toggled=WyI6YmFsbGVyaW5hLWNsaTp0ZXN0LTYiXQ

Reply:
I will enable the tests once more and see if the pipeline fails

Reply:
@hasithaa these tests fail on CI. 
I think they were simply not executed before, since one test was calling `System.exit()`, but I will try to fix these test cases.
Build Scan: https://scans.gradle.com/s/4hiwgkxjcg74w/tests/overview?outcome=FAILED